requirements.txt

```diff
@@ -0,0 +1,4 @@
+Django>2.1, <2.2
```

Comment:
It should be `Django =>, <2.2`

Reply:
So this way it refers directly to the version specified in the .lock file, correct?

Reply:
Sorry, I see that I have not write it correctly last time. It should be `Django => 2.1, < 2.2`